File: Java/1651-1700/1672. Richest Customer Wealth.java
// https://leetcode.com/problems/richest-customer-wealth/

class Solution {
    public int maximumWealth(int[][] accounts) {
        int Rich = 0;

        for(int[] amount : accounts){
            int current = 0;
            for(int balance : amount){
                current += balance;
            }
            Rich = (Rich < current) ? current : Rich;
        }

        return Rich;
    }
}

File: Javascript/1651-1700/1672. Richest Customer Wealth.js
// https://leetcode.com/problems/richest-customer-wealth/

var maximumWealth = function(accounts) {
    let Rich = 0;

    for(const [...amount] of accounts){
        let current = 0;
        for(let balance of amount){
            current += balance;
        }
        Rich = (Rich < current) ? current : Rich;
    }

    return Rich;
};
